src/assets/styles/projects copy.scss

```
@@ -1,216 +1,216 @@
-    }
-  }
+	.projects {
+		padding: 1rem;
+	}
+	.project {
+		&__tags {
+			flex-direction: row;
+			top: 0;
+			left: 0;
+		}
+		// &__header {
```

Stylelint (17.4.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)